integration_tests/sdk/setup_integration.py

```diff
@@ -164,9 +168,20 @@ def _setup_postgres_db():
     _execute_command(["aqueduct", "install", "postgres"])
 
 
+def _setup_mysql_db():
+    print("Setting up MySQL database ...")
+    _execute_command(["aqueduct", "install", "mysql"])
+
+
 def _setup_relational_data(client: Client, db: RelationalDBIntegration) -> None:
     # Find all the tables that already exist.
-    existing_table_names = set(db.list_tables()["tablename"])
+    print('inside _setup_relational_data')
+    print('databaseTables: ' + db.list_tables())
+    # For MYSQL, this field is called TABLE_NAME
+    # Inside list_tables there is a query to get the talbes, need to do the query of table_name AS tablename to work here
+    # Looks like the field here is called TABLE_NAME
+    existing_table_names = set(db.list_tables()["TABLE_NAME"])
```

Comment:
Source of the problem!!!!!!

MySQL is TABLE_NAME, while PG is tablename. Just need to use an 'AS' statement to get this to be the correct key.

Tried to edit the query in list_tables, but I appear to not be editing the right place as none of my print statements in list_tables get hit 😕